### DISCUSSION TITLE
Purchased authentication key failed

### DISCUSSION THREAD

**Discussion by tarsoalvarenga** [yuezk/GlobalProtect-openconnect#369] [General]
Good afternoon Devs!
I recently purchased a lifetime license from you and I have only activated this app on 2 desktops.

I have already encountered a failure 3 times that informs me that my license is no longer valid and therefore, I am forced to reactivate it within the app.

But now it is informing that the key I am using is activated on 3/5 computers and I only have 2 desktops that have this key activated.

Can you help with this problem?

![Sem título](https://github.com/yuezk/GlobalProtect-openconnect/assets/97611311/e7eb475c-8999-46fc-bce4-0dc8dd2f0785)

**Comment by yuezk** [comment]
Hi @tarsoalvarenga Sorry for the inconvenience. This could be a network problem.

Anyway, you can manage your license activations on [My Orders](https://app.lemonsqueezy.com/my-orders). Feel free to let me know if you have any questions. Thanks.
